lnwallet/btcwallet/btcwallet.go

```diff
@@ -431,42 +431,61 @@ func (b *BtcWallet) PublishTransaction(tx *wire.MsgTx) error {
 	if err := b.wallet.PublishTransaction(tx); err != nil {
 		switch b.chain.(type) {
 		case *chain.RPCClient:
+			// Output was already spent.
 			if strings.Contains(err.Error(), "already spent") {
-				// Output was already spent.
 				return lnwallet.ErrDoubleSpend
 			}
+
+			// Output was already spent.
 			if strings.Contains(err.Error(), "already been spent") {
-				// Output was already spent.
 				return lnwallet.ErrDoubleSpend
 			}
+
+			// Transaction is spending either output that is
+			// missing or already spent.
 			if strings.Contains(err.Error(), "orphan transaction") {
-				// Transaction is spending either output that
-				// is missing or already spent.
+				return lnwallet.ErrDoubleSpend
+			}
+
+			// RBF enabled transaction was rejected for whatever
+			// reason.
+			if strings.Contains(err.Error(), "replacement transaction") {
```

Comment:
Heh, good thing the errors returned always include this, makes string matching a bit simpler.